README.md

```diff
@@ -1,9 +1,10 @@
 <p align="center">
   <h1 align="center">fqm-execution</h1>
-  <p align="center">
-    Library for calculating FHIR-based Electronic Clinical Quality Measures (eCQMs) written in Clinical Quality Language (CQL)
-    <br/>
-  </p>
+<div align="center">
+
+Library for calculating HL7® FHIR® standard<sup id="fn-1">[\[1\]](#fnref-1)</sup>-based Electronic Clinical Quality Measures (eCQMs) written in Clinical Quality Language (CQL)
```

Comment:
```suggestion
Library for calculating Electronic Clinical Quality Measures (eCQMs) written in Clinical Quality Language (CQL) using the HL7® FHIR® standard<sup id="fn-1">[\[1\]](#fnref-1)</sup>
```
The "-based" is kind of awkward now. What do you think of changing it around like this?

Reply:
It seems like "-based" is what is used in almost all of our other PRs, but we will have to update those to use the trademark too eventually, so I can happy to make this change.